RELEASE_NOTES.md

```diff
@@ -310,14 +445,6 @@ include the new sigmoid module.  Options are now as follows:
 - Section headers have been added to the "sort by" drop-down in the
   top panel (files, times, etc).
 
-- Shortcuts assigned to presets or styles will now be shown when
-  hovering over them in the relevant menu.
-
-- Long-left-clicking a preset will now keep the preset menu open so
-  you can quickly switch between several presets to visualise the
-  effect. You can also scroll over the preset button to switch to
-  previous/next presets (like you already could using shortcuts).
-
 - When the crop module receives focus and switches to an uncropped
```

Comment:
github doesn't allow me to comment on an unchanged line below, so...
"The crop module, which shows the full image to facilitate making
  adjustments, will no longer trigger a pipe recalculation" feels faintly repetitive when all put together so might be changed to something like
"While the crop module shows the full image to facilitate making
  adjustments, it will no longer trigger a pipe recalculation"

Reply:
resolved in the latest commit